common/sgx/switchless.edl

```diff
@@ -22,6 +22,9 @@ enclave
             [user_check] struct _host_worker_context* host_worker_contexts,
             uint64_t num_host_workers);
 
+        public void oe_switchless_enclave_worker_thread_ecall(
```

Comment:
Why is there an API without _sgx_ in the name, in an SGX-specific file?
This at least requires a // comment.
(E.g., if this API is meant to be TEE-agnostic and should be moved out of this file in the future, then say so)

Reply:
Good point. Added `oe_sgx_` prefix to all the functions in that EDL.